src/Bling.js

```diff
@@ -551,7 +551,18 @@ class Bling extends Component {
             slotSize,
             this.viewableThreshold
         );
-        if (inViewport) {
+
+        const isHidden = () => {
+            const el = ReactDOM.findDOMNode(this);
+            return el.offsetParent === null || el.style.display === "none";
+        };
+
+        if (inViewport && isHidden() === false) {
+            console.log(
```

Comment:
Of course we should remove this log before deployment...

Reply:
yup...i missed that and just removed